erpcgen/src/templates/c_coders.template

```diff
@@ -63,13 +63,16 @@ if (({$info.sizeTemp} <= {$info.maxSize}) && ({$info.dataTemp} != NULL))
 {% if source == "server" || info.useMallocOnClientSide == true %}
 {$indent}{$info.name} = (uint8_t *) erpc_malloc({$info.maxSize} * sizeof(uint8_t));
```

Comment:
Can we not  do malloc at all if the size is zero?
Why the malloc is of max size and not sizeTemp?